atoma-auth/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![allow(clippy::doc_markdown)]
 #![allow(clippy::cast_sign_loss)]
 #![allow(clippy::cast_possible_wrap)]
+#![allow(clippy::module_name_repetitions)]
```

Comment:
we shouldn't need to do this if your local rust is `1.84.0` which is the current LTS

Reply:
Removed all of instances of this